index.js

```diff
@@ -1,6 +1,73 @@
 module.exports = {
-    gltfPipeline : require('./lib/gltfPipeline'),
-    readGltf : require('./lib/readGltf'),
-    writeBinaryGltf : require('./lib/writeBinaryGltf'),
-    writeGltf : require('./lib/writeGltf')
-};
+    AccessorReader : require('./lib/AccessorReader.js'),
```

Comment:
Are you sure you want to expose all of this?  That is _a lot_ of API surface area.  We learned a very hard lesson in Cesium to only expose what is needed to keep the maintain cost way down.

Please only expose what is currently needed and make sure the public API has full reference doc that can be generated like we do for Cesium.  Don't worry about the generated doc's styling or hosting right now.